app/scripts/proactive/model/Task.js

```diff
@@ -347,6 +340,25 @@ define(
                     },
                     confirmDelete: 'You are about to remove a Selection script.'
                 },
+                "Fork": {
+                    type: "Checkbox",
+                    fieldAttrs: {
+                        // The Fork Execution Environment begins the Fork Environment tab, 'data-tab',
+                        // everything which comes after this tab is included in it, if no new 'data-tab'
+                        // is defined.
+                        "data-tab": "Fork Environment",
+                        "data-tab-help": "Fork environment is a new customisable JVM started to only run the task it belongs to. Also specify how to start this JVM, like in a Docker container for example.",
+                        'placeholder': '@attributes->fork',
+                        "data-help": 'Executes the task in a forked JVM, when it is false, all the other fork environment configurations are not taken into account.'
+                    }
+                },
+                "Run as me": {
+                    type: "Checkbox",
+                    fieldAttrs: {
+                        'placeholder': '@attributes->runAsMe',
+                        "data-help": 'Executes the task under your system account.'
```

Comment:
in the help, you should add that this setting implies as well fork

Reply:
Done. Added: ", it also implies the task is executed in a forked JVM."